pkg/controller/s3/bucket/accelerateConfig.go

```diff
@@ -69,6 +73,10 @@ func NewAccelerateConfigurationClient(client s3.BucketClient) *AccelerateConfigu
 func (in *AccelerateConfigurationClient) Observe(ctx context.Context, bucket *v1beta1.Bucket) (ResourceStatus, error) {
 	external, err := in.client.GetBucketAccelerateConfigurationRequest(&awss3.GetBucketAccelerateConfigurationInput{Bucket: aws.String(meta.GetExternalName(bucket))}).Send(ctx)
 	if err != nil {
+		// Short stop method for requests in a region without Acceleration Support
+		if !s3.MethodNotSupported(err) && !s3.ArgumentNotSupported(err) {
```

Comment:
```suggestion
		if s3.MethodNotSupported(err) || s3.ArgumentNotSupported(err) {
```

I think that's what you want to achieve, right? Skip if error is of one of those kinds?